lib/galaxy/authnz/custos_authnz.py

```diff
@@ -392,9 +392,9 @@ def _validate_nonce(self, trans, nonce_hash):
 
     def _load_config_for_cilogon(self):
```

Comment:
Seems that this function is now identical to `_load_config_for_keycloak(self):`. Maybe a chance to reduce code duplication?

Reply:
for refactoring, please see here: 
https://github.com/galaxyproject/galaxy/pull/16497